meta-xt-domx/recipes-core/dbus/dbus-cxx_0.10.0.bb

```diff
@@ -6,7 +6,7 @@ DEPENDS = "boost dbus libsigc++-2.0"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
 
-SRC_URI = "git://github.com/dbus-cxx/${PN}.git;nobranch=1;tag=${PV}"
+SRC_URI = "https://github.com/dbus-cxx/${PN}.git;nobranch=1;tag=${PV}"
```

Comment:
I suspect that this change doesn't work.
As I understand, `git:` prefix is used to select git-fetcher.
To use git fetcher with https protocol we need to use `protocol=https` as mentioned below in this commit.

Reply:
Fixed